k8s/sample-pv-yamls/blockdeviceclaim.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: openebs.io/v1alpha1
+kind: BlockDeviceClaim
+metadata:
+  name: test-blockdeviceclaim
+  namespace: openebs
+spec:
+  driveType: ""
+  blockDeviceName: "blockdevice-1234"
+  hostName: ""
```

Comment:
Why is this bank? Can this be removed or commented

Reply:
Added comments